tutorials/napps/development_environment_setup.rst

```diff
@@ -158,8 +158,15 @@ and run:
 .. code-block:: console
 
   $ source test42/bin/activate
+  $ pip install filelock==3.0.12
```

Comment:
This will surely get outdated with time. Shouldn't we tell users to `pip install -r requirements/dev.txt` from the NApp source directory?

Reply:
I was thinking about it and I agree with @hdiogenes. My suggestion would be to add one more step to clone kytos/storehouse from GitHub, install the NApp and their requests/dev.txt. So as one consequence, the kytos/storehouse would be removed from the next step (the installation using kytos napps install command). The note ".. NOTE:: The `filelock` package" could explain why the installation of kytos/storehouse are being done in a different way in this tutorial.

Reply:
Solved in https://github.com/kytos/documentation/pull/106/commits/8de12b8f1b8e7bd0705c1c4ef8d2536cd359345e